src/js/modules/core/actions.js

```diff
@@ -20,35 +16,41 @@ export const closeSidebar = () => ({
 
 export const fetchAllData = () => {
   return dispatch => {
-    return new Promise((resolve, reject) => {
-      resolve(dispatch(fetchSections()));
-    })
-      .then(response => {
-        dispatch(fetchComments());
-      })
-      .then(response => {
-        dispatch(fetchMedia());
-      })
-      .then(response => {
-        dispatch(fetchRoles());
-      })
-      .then(response => {
-        dispatch(fetchUsers());
-      })
-      .then(response => {
-        dispatch(fetchUserRoles());
-      })
-      .then(response => {
-        dispatch(fetchRoles());
-      })
-      .then(response => {
-        dispatch(fetchAuthorships());
-      })
-      .then(response => {
-        dispatch(fetchArticles());
-      })
-      .then(response => {
-        dispatch(fetchOutquotes());
+    dispatch({type: t.FETCH_INIT_DATA_PENDING})
+    axios
+      .get(`${STUY_SPEC_API_URL}/init`, STUY_SPEC_API_HEADERS)
+      .then(response => {
+        throwForEmptySlices(response.data);

```

Comment:
validateSlices?

Reply:
Modeling this name off of Python requests' `response.raise_for_status()`.

Reply:
Well don't